src/Components/Facility/ConsultationForm.tsx

```diff
@@ -1022,29 +1020,24 @@ export const ConsultationForm = (props: any) => {
                 m<sup>2</sup>
               </div>
               {/* End of Telemedicine fields */}
-              <div className="mt-4 flex justify-between">
-                <Button
-                  color="default"
-                  variant="contained"
+              <div className="mt-4 sm:flex grid sm:justify-between">
+                <button
+                  className="btn text-gray-900 bg-gray-200 border border-gray-300 focus:outline-none hover:bg-gray-400  font-medium  text-sm px-5 py-2.5 mt-2"
```

Comment:
Use `btn-default` class for the secondary button
```suggestion
                  className="btn btn-default mt-2"
```